### ISSUE TITLE
version testing

### ISSUE THREAD

**Issue by ClayBirkett** [plantbreeding/IPK-BrAPI-Validator#42]
The BrAPI Validator at http://webapps.ipk-gatersleben.de/brapivalidator/ does its automated weekly test using v1.2. Can this be changed to v1.3? Better yet, can it read the version from the calls endpoint and test at the highest version supported?

**Comment by langeipk** [comment]
Indeed, the  BrAPI Validator does  not set the BrAPI version for the scheduled test dynamical from the results of the calls call. Rather the the property "testedVersion" in config.properties is used to set one version for all endpoints.

This would be a nice add-on to support multiple versions for each endpoint or alternatively the most recent one, as mentioned by @ClayBirkett.

So pull requests are welcome.
